src/instructlab/model/chat.py

```diff
@@ -348,6 +358,7 @@ def __init__(
         self.log_file = log_file
         self.greedy_mode = greedy_mode
```

Comment:
Can we get rid of greedy_mode entirely, or make a note to get rid of it next release. Being able to set the temperature in a flag and a config variable should be enough.